src/library-authoring/studio-header/StudioHeader.jsx

```diff
@@ -50,6 +50,7 @@ const StudioHeader = ({ intl, library }) => {
                     </Dropdown.Toggle>
                     <Dropdown.Menu className="p-4 mt-1 fade">
                       <Dropdown.Item className="p-0" as={Link} to={ROUTES.Detail.EDIT_SLUG(library.id)}>{intl.formatMessage(messages['library.header.settings.details'])}</Dropdown.Item>
+                      <Dropdown.Item className="p-0" as={Link} to={ROUTES.Detail.ACCESS_SLUG(library.id)}>{intl.formatMessage(messages['library.header.settings.access'])}</Dropdown.Item>
```

Comment:
**Deferrable:**

If I click on this menu item with View access, it loads the access page and then immediately kicks me back to the detail view.

I don't have official guidance from UX on this yet, but I imagine that for View-only users, we'll want to either (1) show the access page, even if it's read-only, (2) disable the User Access menu item, or (3) hide the User Access menu item.

Reply:
Added a comment to the README for now.